cobigen-cli/cli/src/main/java/com/devonfw/cobigen/cli/commands/GenerateCommand.java

```diff
@@ -80,6 +84,13 @@ public class GenerateCommand extends CommandCommons {
   @Option(names = { "--templates", "-t" }, split = ",", description = MessagesConstants.TEMPLATES_OPTION_DESCRIPTION)
   List<String> templates = null;
 
+  /**
+   * allows usage of the old monolithic template structure instead of the new template sets structure.
+   */
+  @Option(names = { "--force-monolithic-configuration",
+  "--force-mc" }, description = MessagesConstants.FORCE_MONOLITHIC_CONFIGURATION)
+  boolean forceMonolithicConfiguration;
+
```

Comment:
Can be removed is already defined in the supertype commandCommons

Reply:
not the same